src/components/Book.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+export default function User(props) {
+  const book = props;
+  return (
+    <>
+      <li className="buttons" key={book.id}>
+        {book.category}
+        {book.name}
+        {book.author}
+        <button type="button">Comments</button>
```

Comment:
- [ ] I can't able to see any book in the UI. Make sure you are sending the props in the correct way. I can only see the buttons and not any book data. Please fix it.


![Screenshot from 2021-10-05 22-07-44](https://user-images.githubusercontent.com/22434660/136067369-6185457b-1979-4321-a08e-f0e6782855c0.png)

Reply:
Thank you for suggesting what to do. I fixed the problem and now it is showing books.